.github/workflows/google_api_availability_android.yaml

```diff
@@ -40,9 +40,9 @@ jobs:
         run: flutter pub get
         working-directory: ${{env.source-directory}}
 
-      # Run Flutter Format to ensure formatting is valid
-      - name: Run Flutter Format
-        run: flutter format --set-exit-if-changed .
+      # Run dart Format to ensure formatting is valid
+      - name: Run dart Format
+        run: dart format .
```

Comment:
I think we still need the `--set-exit-if-changed` flag, as this will ensure the `dart format` command will return with an error code and cause the build step to fail.